celery/canvas.py

```diff
@@ -1415,20 +1423,27 @@ def apply(self, args=None, kwargs=None,
         )
 
     @classmethod
-    def __descend(cls, sig_obj):
+    def _descend(cls, sig_obj):
```

Comment:
Why the name change?

Reply:
The method gets mangled with the old name and we need to call it from the `group` class now rather than just `chord`.